test/vector-tile.composite.test.js

```diff
@@ -313,14 +313,14 @@ describe('mapnik.VectorTile.composite', function() {
                 assert.equal(0,compare_to_image(im2,expected_file2));
                 vtile3.composite([vtile2],{reencode:true}, function(err) {
                     if (err) throw err;
-                    assert.equal(vtile3.getData().length,54781);
+                    assert.equal(vtile3.getData().length,54837);
                     assert.deepEqual(vtile3.names(),["water","admin"]);
                     vtile3.render(map,new mapnik.Image(256,256),function(err,im) {
                         if (err) throw err;
                         assert.equal(0,compare_to_image(im,expected_file));
                         vtile4.composite([vtile2],{reencode:true,max_extent:world_clipping_extent}, function(err) {
                             if (err) throw err;
-                            assert.equal(vtile4.getData().length,54522);
+                            assert.equal(vtile4.getData().length,54576);
                             assert.deepEqual(vtile4.names(),["water","admin"]);
```

Comment:
Wagyu tends to be able to keep more data then the angus clipper did in general for attempting to put valid geometry into the data. Therefore, it has been fairly consistent that more data has been added.